include/ftsh.h

```diff
@@ -0,0 +1,32 @@
+#ifndef FTSH
+#define FTSH
```

Comment:
The variable naming scheme here is not standardized across source files. Please use `_FULLTEXT_NAME_H_` across the board (that is, this would be `_FULLTEXT_FILE_H_`).